source/common/common/win32/event_logger_impl.h

```diff
@@ -0,0 +1,21 @@
+#pragma once
+
+#include "common/common/base_logger.h"
+
+namespace Envoy {
+namespace Logger {
+
+#define GENERATE_WIN32_NATIVE_LOGGER(X) WindowsEventLogger(#X),
+
+/**
+ * Logger that uses `spdlog::sinks::win_eventlog_sink`.
+ */
+class WindowsEventLogger : public Logger {
+public:
+  WindowsEventLogger(const std::string& name);
+
+  friend class ServiceBase;
```

Comment:
Why is this needed?

Reply:
I thought it was needed because of a TODO, but it turns out it is not. I removed it in It is not needed, I thought it was needed as I was experimenting